scripts/_check_docs_utils.py

```diff
@@ -0,0 +1,718 @@
+# coding: utf-8
```

Comment:
Not keen on mixing up third-party code in our regular codebase like this.

Can we put this in third_party instead and refer to it somehow, swapping out methods (just for the duration of the linter run) if needed?

Reply:
I have modified `.gitignore` to allow `_check_docs_utils.py` which has been placed in `python-custom-checks`. Thanks!